.github/workflows/ci.yml

```diff
@@ -68,6 +68,12 @@ jobs:
       - ci-backend
       - ci-frontend
     steps:
+      -
+        name: Set gchr repository name
+        id: set-ghrc-repository
```

Comment:
Believe `ghcr` typo x2 here (and same for refs to this)

Reply:
Should be fixed